### DISCUSSION TITLE
코스메틱 규칙에서 도메인 숫자(이름)에 와일드카드 기호 지원

### DISCUSSION THREAD

**Discussion by hsp1020** [List-KR/community#64] [AdGuard]
국내 토렌트, 동영상 사이트 중 주기적으로 도메인 뒤의 숫자를 바꾸는 사이트들이 다수 있습니다.(newtoki, torrentqq  등)

유니콘 PRO 같은 경우에는 
코스메틱 규칙에서 도메인 숫자(이름)에 와일드카드 기호 지원(예 : newtoki*.com###.hd_pop)
으로 도메인 뒤의 숫자가 바뀌더라도 와일드카드만으로 별도 코드 수정없이 지속적으로 광고 차단이 가능합니다.

반면 애드가드는 아직 해당 기능을 지원하지 않는것 같더군요
실제로 List-KR의 필터링 로그를 살펴보니

newtoki335.com,newtoki336.com,newtoki337.com,newtoki338.com,newtoki339.com,newtoki340.com,newtoki341.com,newtoki342.com,newtoki343.com,newtoki344.com,newtoki345.com,newtoki346.com###hd_pop

이런식으로 숫자를 일일히 바꾸어 나열하는 방식으로 광고를 차단하고 있더라고요

애드가드 측에 코스메틱 규칙에서 도메인 숫자(이름)에 와일드카드 기호 지원 하도록 요청 할 수 있을까요? 
이 부분이 반영되면 List-KR 필터도 일부 코드가 간소화 될 수 있기에 분명 필요한 것 같습니다.

**Comment by piquark6046** [comment]
이미 있는데요.
https://adguard.com/kb/general/ad-filtering/create-own-filters/#non-basic-url-modifier
AdGuard for iOS는 미지원이고 AdGuard 브라우저 확장 프로그램은 지원 예정이라서 사용하지 않았습니다.

**Reply by hsp1020** [reply]
[$url=/\manatoki[0-9]+\\.net^/]###hd_pop
작성해주신 내용 기반으로 작성해서 사용자 규칙에 추가했지만
마나토끼 접속시 최초 팝업창은 여전히 차단되지 않더라고요(윈도우, 안드로이드용 모두)
참고로 해당 팝업창은 List-KR 필터로는 차단되지 않은 상태입니다.

**Reply by piquark6046** [reply]
```adblock
[$url=/manatoki[0-9]+\.net/]###hd_pop
```
위 규칙으로도 안되나요?

**Reply by hsp1020** [reply]
작성해주신대로 하니 되네요 감사합니다.
